### ISSUE TITLE
Suggestion to add server-toggle configuration (Crash fix)

### ISSUE THREAD

**Issue by Pepsied-5229** [19Phin/Multiccount#1]
I like to use this mod on singleplayer, but whenever I join a server that doesn't have the mod installed, my game crashes. I was hoping to see a server-enabled toggle or maybe detection for when a server doesn't have it installed so it wouldn't crash.

**Comment by DialingSpoon527** [comment]
Sorry about that, I never thought to check how it would behave on a server without it installed server-side should be a quick fix.

**Comment by DialingSpoon527** [comment]
@Pepsied-5229 Actually I cant seem to replicate, what version are you using?

**Comment by Pepsied-5229** [comment]
As much as it pains me, after trying to get a fresh crash log, I can say that it's not crashing anymore. I think it may have been a mod conflict then, because I just changed a bunch of mods around. The crash stopped happening after singularly disabling Multiccount so I assumed that the mod crashed on unsupported servers. For additional context, I'm running version 1.1.0 on 1.20.4 Fabric 1.15.3.0; API: 0.91.3 with about 178 other mods. I had to use an older version of fabric & API that was used for 1.20.2 because of mod conflicts. And of course, I cleared the old logs... I'm very sorry about this, I cannot replicate either.
